### DISCUSSION TITLE
Does ParallelComputingInterface support time_series mode?

### DISCUSSION THREAD

**Discussion by paulf81** [NatLabRockies/floris#705] [Q&A]
Question for @rafmudaf  and @Bartdoekemeijer , does ParallelComputingInterface support time_series mode?  Is it just a matter of passing the flag through, or would this break something I'm missing?

**Comment by Bartdoekemeijer** [comment]
`ParallelComputingInterface` splits the FLORIS object over the wind direction and optionally over the wind speed. My understanding is that `time_series=True` mode expands all the wind conditions along the direction of the wind direction dimension. In principle, this _should_ carry over. However, when writing up `ParallelComputingInterface`, I did not pay any attention to specific syntax required for `time_series=True` mode, so my guess is that it won't work out of the box. My feeling is that it should be a simple lift to get it fixed, though.

**Comment by paulf81** [comment]
Ok sounds good, I'm going to open an issue, thanks @Bartdoekemeijer !
